VirtoCommerce.Storefront/Controllers/AccountController.cs

```diff
@@ -360,5 +359,17 @@ public ActionResult ForgotPassword()
                 return View("customers/account", WorkContext);
             }
         }
+
```

Comment:
Better write the extension method for array calims names  `ClaimsPrincipal.FindFirstValue(string[] claimTypes, string defaultValue)` istead of made separate methods.